config/config.ini

```diff
@@ -79,3 +79,10 @@ TransactionTax = 0.15
 [Betting]
 RollCases = 66, 90, 99, 100
 RollReturns = 0, 2, 4, 10
+
+[Roles]
+Pronouns = She/Her, He/Him, They/Them
```

Comment:
gonna hav eto clear all of these categories with mods sry

Reply:
Yep, goal is to get some in here so that we can test as the default config. The actual config will differ a lot.